src/App.svelte

```diff
@@ -13,10 +13,9 @@
 
   const options = get(optionsStore)
   const paymentSessionIdUrlKey = "orderId"
+  const toonieRedirectUrl = "https://www.toonieglobal.com/"
```

Comment:
Do we want to make this an env variable? Or will it always be this one?
```suggestion
  const redirectUrl = "https://www.toonieglobal.com/"
```

Reply:
I'm gonna add a comment because it is a temporary solution we decided with Gian to not allow the user to see the data about the payment when he comes back after finishing the payment